banyand/measure/measure_query.go

```diff
@@ -54,10 +54,25 @@ type Measure interface {
 	GetSchema() *databasev1.Measure
 	GetIndexRules() []*databasev1.IndexRule
 	GetInterval() time.Duration
+	BuildSchema(other Measure) *databasev1.Measure
 }
 
 var _ Measure = (*measure)(nil)
 
+func (s *measure) BuildSchema(other Measure) *databasev1.Measure {
```

Comment:
According to our last discussion, we should build topN schema with entity names,

https://github.com/apache/skywalking-banyandb/blob/d722db6bcd52554ef429961600d617d28d561ae5/banyand/measure/metadata.go#L204-L220

The actual entity names written for the TopN data point can be found,

https://github.com/apache/skywalking-banyandb/blob/d722db6bcd52554ef429961600d617d28d561ae5/banyand/measure/measure_topn.go#L233-L247